app/views/pages/about.html.erb

```diff
@@ -7,12 +7,12 @@
 
   <h3 class=about_h3>Project Overview</h3>
     <p class="about_p">The Archives Unleashed Cloud, referred to as AUK, is an open source cloud-based analysis tool that helps researchers and scholars conduct web archive analysis.</p>
-    <p class="about_p">AUK is a component of the <a href="https://archivesunleashed.org" target="_blank">Archives Unleashed Project</a>, which aims to allow researchers, scholars, librarians and archivists the ability to access and investigate their web archival collections. As accessibility is a main priority of the project, AUK supports this goal by providing a web-based front end for users to access the <a href="https://archivesunleashed.org/aut/" target="_blank">Archives Unleashed Toolkit</a>.</p>
+    <p class="about_p">AUK is a component of the <%= link_to('Archives Unleashed Project', 'https://archivesunleashed.org', target: '_blank') %>, which aims to allow researchers, scholars, librarians and archivists the ability to access and investigate their web archival collections. As accessibility is a main priority of the project, AUK supports this goal by providing a web-based front end for users to access the <%= link_to('Archives Unleashed Toolkit.', 'https://archivesunleashed.org/aut/', target: '_blank') %>
```

Comment:
Super pedantic: could we move the period after "Archives Unleashed Toolkit" outside of the link? (just for consistency with rest of site)

Reply:
Thanks for the review!